lib/jmap/core/method/response/query_response.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/method/method_response.dart';
+import 'package:jmap_dart_client/jmap/core/state.dart';
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
+
+abstract class QueryResponse extends ResponseRequiringAccountId {
```

Comment:
how about `limit`, `position`?

Reply:
It is not a generic attribute. So will implement in specific queries

Reply:
https://jmap.io/spec-core.html#query